[0.124s] Invoking command in '/home/robotica/taller2_grupo15/build/services': CMAKE_PREFIX_PATH=/home/robotica/taller2_grupo15/install/mi_robot_15:/opt/ros/humble /usr/bin/cmake /home/robotica/taller2_grupo15/src/services -DCMAKE_INSTALL_PREFIX=/home/robotica/taller2_grupo15/install/services
[0.150s] -- Found ament_cmake: 1.3.3 (/opt/ros/humble/share/ament_cmake/cmake)
[0.700s] -- Found geometry_msgs: 4.2.3 (/opt/ros/humble/share/geometry_msgs/cmake)
[0.780s] -- Found rosidl_generator_c: 3.1.4 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[0.798s] -- Found rosidl_adapter: 3.1.4 (/opt/ros/humble/share/rosidl_adapter/cmake)
[0.816s] -- Found rosidl_generator_cpp: 3.1.4 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[0.844s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[0.906s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[1.273s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[1.779s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[1.784s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name
[1.785s]   services --arguments-file
[1.785s]   /home/robotica/taller2_grupo15/build/services/rosidl_adapter__arguments__services.json
[1.789s]   --output-dir
[1.792s]   /home/robotica/taller2_grupo15/build/services/rosidl_adapter/services
[1.793s]   --output-file
[1.793s]   /home/robotica/taller2_grupo15/build/services/rosidl_adapter/services.idls)
[1.793s]   returned error code 1:
[1.793s] 
[1.795s]   Traceback (most recent call last):
[1.795s] 
[1.795s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[1.795s]       return _run_code(code, main_globals, None,
[1.795s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[1.795s]       exec(code, run_globals)
[1.797s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[1.798s]       sys.exit(main())
[1.798s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[1.798s]       abs_idl_file = convert_to_idl(
[1.798s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 24, in convert_to_idl
[1.798s]       return convert_srv_to_idl(
[1.798s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/srv/__init__.py", line 28, in convert_srv_to_idl
[1.798s]       srv = parse_service_string(package_name, input_file.stem, content)
[1.809s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 850, in parse_service_string
[1.809s]       raise InvalidServiceSpecification(
[1.809s] 
[1.809s]   rosidl_adapter.parser.InvalidServiceSpecification: Could not find separator
[1.810s]   '---' between request and response
[1.810s] 
[1.810s] Call Stack (most recent call first):
[1.810s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[1.810s]   CMakeLists.txt:15 (rosidl_generate_interfaces)
[1.810s] 
[1.810s] [0m
[1.810s] -- Configuring incomplete, errors occurred!
[1.810s] See also "/home/robotica/taller2_grupo15/build/services/CMakeFiles/CMakeOutput.log".
[1.835s] Invoked command in '/home/robotica/taller2_grupo15/build/services' returned '1': CMAKE_PREFIX_PATH=/home/robotica/taller2_grupo15/install/mi_robot_15:/opt/ros/humble /usr/bin/cmake /home/robotica/taller2_grupo15/src/services -DCMAKE_INSTALL_PREFIX=/home/robotica/taller2_grupo15/install/services
